src/main/java/dev/sefiraat/cultivation/implementation/listeners/CustomPlacementListener.java

```diff
@@ -79,6 +81,14 @@ public void onBoneMeal(@Nonnull BlockFertilizeEvent event) {
         }
     }
 
+    @EventHandler
+    public void onSnowmanBlockForm(@Nonnull EntityBlockFormEvent event) {
+        if (event.getEntity() instanceof Snowman) {
```

Comment:
Question, wouldn't we want to remove our item for ANY mob forming a block over our items?